robotmbt/suiteprocessors.py

```diff
@@ -88,41 +88,53 @@ def flatten(self, in_suite: Suite) -> Suite:
         out_suite.suites = []
         return out_suite
 
-    def process_test_suite(self, in_suite: Suite, *, seed: any = 'new', graph: str = '') -> Suite:
+    def process_test_suite(self, in_suite: Suite, *, seed: any = 'new', graph: str = '',
+                           to_json: str = 'false', from_json: str = 'false') -> Suite:
         self.out_suite = Suite(in_suite.name)
         self.out_suite.filename = in_suite.filename
         self.out_suite.parent = in_suite.parent
         self._fail_on_step_errors(in_suite)
         self.flat_suite = self.flatten(in_suite)
 
-        for id, scenario in enumerate(self.flat_suite.scenarios, start=1):
-            scenario.src_id = id
-        self.scenarios = self.flat_suite.scenarios[:]
-        logger.debug("Use these numbers to reference scenarios from traces\n\t" +
-                     "\n\t".join([f"{s.src_id}: {s.name}" for s in self.scenarios]))
+        if from_json != 'false':
+            with open(f"json/{from_json}.json", "r") as f:
```

Comment:
please don't hard code paths in multiple files. Can we have some central place where we store this path? This makes it easy to configure in the future. Code smell.